lib/cf_obs_binary_builder/buildpacks/base_buildpack.rb

```diff
@@ -44,6 +44,19 @@ def update_source_tarball
 
     system("tar -xf v#{version}.tar.gz")
 
+    if File.file?(File.join(tarball_dir,'go.mod'))
+      puts "Buildpack contains go.mod. Generating vendor/ and bundling buildpack-packager"
+
+      # We have to fetch buildpack-packager as
+      # `go mod vendor` filters out non-modules.
+      if !system("cd #{tarball_dir} && wget https://raw.githubusercontent.com/cloudfoundry/libbuildpack/master/packager/buildpack-packager/main.go -O buildpack-packager.go")
+        raise "Could not download buildpack-packager"
+      end
+      if !system("cd #{tarball_dir} && go mod vendor")
```

Comment:
I would add a check for go >= 1.11 with a proper error message.

Reply:
Fixed in https://github.com/SUSE/cf-obs-binary-builder/pull/3/commits/3b4977ef374bfd5c4c66c48500d06e530a8f4cbb